### PR TITLE
[16.0][OU-ADD] note_pad: Nothing to do

### PR DESCRIPTION
**Pull request by daiduongnguyen-odoo**
Obsolete module, use directly in odoo itself

Here is the PR that merged the feature `https://github.com/odoo/odoo/pull/75768`  and here the PR that delete it `https://github.com/odoo/odoo/pull/76467`

### CODE REVIEW

File: docsource/modules150-160.rst [hunk 1]
```diff
@@ -528,7 +528,7 @@ Module coverage 15.0 -> 16.0
 +-------------------------------------------------+----------------------+-------------------------------------------------+
 | note                                            |Nothing to do         |                                                 |
 +-------------------------------------------------+----------------------+-------------------------------------------------+
-| |del| note_pad                                  |                      |                                                 |
+| |del| note_pad                                  |Done                  |Obsolete module, use directly in odoo itself     |
```
Comment (MiquelRForgeFlow) [inline docsource/modules150-160.rst]:
```suggestion
| |del| note_pad                                  |Done                  | Merged into web_editor |
```

File: openupgrade_scripts/apriori.py [hunk 2]
```diff
@@ -45,6 +45,7 @@
     "sale_project_account": "sale_project",
     "website_sale_delivery_giftcard": "website_sale_loyalty_delivery",
     "website_sale_gift_card": "website_sale_loyalty",
+    "note_pad": "web_editor",
```
Comment (MiquelRForgeFlow) [inline openupgrade_scripts/apriori.py]: please, respect alphabetical order

Comment (daiduongnguyen-odoo) [reply openupgrade_scripts/apriori.py]: Oki

**Comment by MiquelRForgeFlow** [comment]
please rebase

**Comment by daiduongnguyen-odoo** [comment]
> please rebase

Done sir, please merge then i will rebase another one :v

**Comment by pedrobaeza** [comment]
This is not the more appropriate one. It should be merged into `note`.

**Pull request merged**